pyintelowl/cli/jobs.py

```diff
@@ -76,8 +77,25 @@ def jobs(ctx: ClickContext, id: int, all: bool, status: str):
     help="sleep interval before subsequent requests (in sec)",
 )
 @click.pass_context
-def poll(ctx: ClickContext, max_tries: int, interval: int):
-    pass
+def poll(ctx: ClickContext, id: int, max_tries: int, interval: int):
+    ctx.obj.logger.info(f"Polling Job [underline blue]#{id}[/]..")
+    with click_spinner.spinner():
+        poll_for_job(ctx, id, max_tries, interval)
+
+
+def poll_for_job(ctx: ClickContext, id: int, max_tries: int, interval: int):
+    poll_result = {}
+    try:
+        for i in range(max_tries):
+            ans = ctx.obj.get_job_by_id(id)
+            if ans["status"].lower() not in ["running", "pending"]:
+                break
+            _display_single_job(ans)
+            Console().clear()
+            time.sleep(interval)
+    except Exception:
+        print("Error in retrieving job!")
```

Comment:
should be `except Exception as e: ctx.obj.logger.error(f"Error in retrieving job: {str(e)}")`